javaparser-core/src/main/java/com/github/javaparser/JavaToken.java

```diff
@@ -51,41 +55,36 @@ public JavaToken(int kind, String text) {
     }
 
     JavaToken(Token token, List<JavaToken> tokens) {
-        Range range = Range.range(token.beginLine, token.beginColumn, token.endLine, token.endColumn);
-        String text = token.image;
-
         // You could be puzzled by the following lines
-        //
+        // 
         // The reason why these lines are necessary is the fact that Java is ambiguous. There are cases where the
         // sequence of characters ">>>" and ">>" should be recognized as the single tokens ">>>" and ">>". In other
         // cases however we want to split those characters in single GT tokens (">").
-        //
+        // 
```

Comment:
Maybe we do not want these extra spaces. Not that it really matters :)

Reply:
Everything gets reformatted because this file is now part of the code generation step. See also #1301 :-)